tomcat-gcp-lib/pom.xml

```diff
@@ -99,13 +99,21 @@
 
   <profiles>
     <profile>
-      <id>datastore-integration-test</id>
+      <id>gcloud-integration-test</id>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-failsafe-plugin</artifactId>
             <version>2.20</version>
+            <executions>
+              <execution>
+                <goals>
+                  <goal>integration-test</goal>
+                  <goal>verify</goal>
```

Comment:
Well, as these tests are dependent of the environment and can have a cost due to the usage of the Datastore, I would think that they should be optional.